geppetto.js/geppetto-ui/package.json

```diff
@@ -116,9 +109,17 @@
     "jest": "^24.9.0",
     "typescript": "^4.3.2"
   },
+  "peerDependencies": {
```

Comment:
Later on, we probably want to do this for more than threejs don't you think?

Reply:
I think we should move everything from dependencies to peerDependencies

Reply:
We did that